index.html

```diff
@@ -1,21 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Git and GitHub introduction</title>
     <link rel="stylesheet" href="styles.css">
 </head>
+
 <body>
-   <h1>
-       Welcome to Git and GitHub introduction project
-   </h1> 
-   <p>
-       Feel free to edit this project and send pull request.
-   </p>
-   <p>
-       If you don't know how, visit this <a href="http://makeapullrequest.com/" target="_blank">link</a>.
-   </p>
+    <h1>
+        Welcome to Git and GitHub introduction project
+    </h1>
+    <p>
+        Feel free to edit this project and send pull request.
+    </p>
+    <p>
+        If you don't know how, visit this <a href="http://makeapullrequest.com/" target="_blank"
+            class="button"><button class="btn">Link</button></a>
```

Comment:
It is not semantically correct to nest `button` element inside `a` element. You can read more about it [here.](https://stackoverflow.com/a/6393863/3865116). As a rule of thumb if the link should navigate to some other page use `a` and if it represents some action that needs to happen on the same page use `button`.

Reply:
> It is not semantically correct to nest `button` element inside `a` element. You can read more about it [here.](https://stackoverflow.com/a/6393863/3865116). As a rule of thumb if the link should navigate to some other page use `a` and if it represents some action that needs to happen on the same page use `button`.

Thank you so much for your help. Learned a lot of things from this experience.